larray/core/array.py

```diff
@@ -3601,7 +3603,7 @@ def rationot0(self, *axes):
 
         for reference, the normal ratio method would produce a warning message and return:
 
-        >>> arr.ratio('a')
+        >>> arr.ratio('a')                                          # doctest: +SKIP
```

Comment:
Is it tested in our unittests ?

Reply:
Not sure about unit tests but it is at least tested in the doctest of the ratio method (this is rationot0).

Reply:
OK